beater/config_test.go

```diff
@@ -84,7 +87,11 @@ func TestConfig(t *testing.T) {
 				WriteTimeout:    4000000000,
 				ShutdownTimeout: 9000000000,
 				SecretToken:     "1234random",
-				SSL:             &SSLConfig{Enabled: &truthy, Certificate: outputs.CertificateConfig{Certificate: "1234cert", Key: "1234key"}},
+				TLS: &tlscommon.ServerConfig{
+					Enabled:     &truthy,
+					CAs:         []string{"./ca.cert.pem"},
+					Certificate: outputs.CertificateConfig{Certificate: "1234cert", Key: "1234key"},
+					ClientAuth:  4}, //4=RequireAndVerifyClientCert
```

Comment:
This needs to be changed to `0`, as libbeat changed the default back to `none`.

Reply:
libbeat now updated

Reply:
7.x is not on the latest libbeat version, that's why `ClientAuth: 4` is ok here. Once we update to the latest, it should be 0 again.